ansible/files/runtimes.json

```diff
@@ -240,6 +240,23 @@
             }
         ]
     },
+    "dotnet": [
+        {
+            "kind": "dotnet:2.2",
+            "default": true,
+            "deprecated": false,
+            "requireMain": true,
+            "image": {
+                "prefix": "openwhisk",
+                "name": "action-dotnet-v2.2",
+                "tag": "latest"
+            },
+            "attached": {
+                "attachmentName": "codefile",
+                "attachmentType": "application/zip"
```

Comment:
```suggestion
                "attachmentType": "application/octet-stream"
```

Reply:
weird that github wont let me commit this suggestion. @shawnallen85 can you try.

Reply:
@chetanmeh can you review this change? 

All of the attachments in the manifest are text/plain which iirc is because we store all blobs as 64 encoded text. So should this also be `plain/text`?

Reply:
Actually is `text/plain`